images/ansible-operator/Dockerfile

```diff
@@ -1,23 +1,67 @@
-# Build the manager binary
-FROM --platform=$BUILDPLATFORM golang:1.19 as builder
+# This Dockerfile defines the base image for the ansible-operator image.
+# It is built with dependencies that take a while to download, thus speeding
+# up ansible deploy jobs.
+
+FROM registry.access.redhat.com/ubi8/ubi:8.8 AS basebuilder
+
+# Install Rust so that we can ensure backwards compatibility with installing/building the cryptography wheel across all platforms
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+ENV PATH="/root/.cargo/bin:${PATH}"
+RUN rustc --version
+
+# Copy python dependencies (including ansible) to be installed using Pipenv
+COPY images/ansible-operator/Pipfile* ./
+# Instruct pip(env) not to keep a cache of installed packages,
+# to install into the global site-packages and
+# to clear the pipenv cache as well
+ENV PIP_NO_CACHE_DIR=1 \
+    PIPENV_SYSTEM=1 \
+    PIPENV_CLEAR=1
+# Ensure fresh metadata rather than cached metadata, install system and pip python deps,
+# and remove those not needed at runtime.
+# pip3~=21.1 fixes a vulnerability described in https://github.com/pypa/pip/pull/9827.
```

Comment:
Probably no longer need this as we've upgraded pip

Reply:
https://github.com/operator-framework/ansible-operator-plugins/pull/5/commits/31da06c117a389e508e24a4e93b45b13a88ce304